spec/support/shared_examples/forms_controller_change_in_hours_behavior.rb

```diff
@@ -4,19 +4,32 @@
   it_behaves_like "form controller base behavior", "change_in_hours"
 
   describe "show?" do
+    let(:report) { create :report, :filled }
+
+    before do
+      member = create :member, report: report
+      change = create :change,
+        change_type: change_type,
+        member: member
+      create :navigator,
+        current_member: member,
+        current_change: change,
+        report: report
+    end
```

Comment:
Is this still needed now that the `:filled` trait is on the `report` factory?

Reply:
Cleaned up all these behavior specs. Thanks.